_events/sr2022/kickstart.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Kickstart
+date: 2021-10-13 13:00:00 Europe/London
```

Comment:
This is today. Is that intentional? It means that this event isn't showing up on the website.

Reply:
Ah damn it. No, no it is not.